app/components/publications/publication/case/contact-person-add-modal.js

```diff
@@ -0,0 +1,96 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { isPresent } from '@ember/utils';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import {
+  ValidatorSet, Validator
+} from 'frontend-kaleidos/utils/validators';
+
+export default class PublicationsPublicationCaseContactPersonAddModalComponent extends Component {
+  @service store;
+
+  @tracked isOpenOrganizationAddModal = false;
+
+  @tracked _organizations;
+
+  @tracked firstName = '';
+  @tracked lastName = '';
+  @tracked email = '';
+  @tracked organization;
+
+  constructor() {
+    super(...arguments);
+    this.loadOrganizations.perform();
+    this.initValidators();
+  }
+
+  get organizations() {
+    return this._organizations?.sortBy('name');
+  }
+
+  get isLoading() {
+    return this.loadOrganizations.isRunning || this.onSave.isRunning;
+  }
+
+  get canCancel() {
+    return !this.onSave.isRunning;
+  }
+
+  get canSave() {
+    return !this.loadOrganizations.isRunning && this.validators.areValid;
+  }
+
+  @task
+  *loadOrganizations() {
+    // TODO This is not ideal, there are currently +- 60 organizations that come from ACM-IDM, they don't have a name
+    // TODO need a better filter, add a boolean to model maybe ?
+    const organizations = yield this.store.query('organization', {
+      page: {
+        size: 200,
+      },
+    });
+    this._organizations = organizations.filter((org) => org.name);
```

Comment:
Zoals je zelf aangeeft in je PR description is dit niet ideaal omdat het aantal organizaties op productie kan oplopen. Ik zou dit dus aanpassen dat de power-select in de template werkt met een search op naam naar de backend. Initieel zou ik slechts een lijst van 10 organizaties inladen. De sortering op naam kan dan ook in de backend gebeuren.

Om de organizaties zonder naam eruit te filteren, heb ik vandaag van @madnificent geleerd dat je volgende filter kan toevoegen in de request naar mu-cl-resources:
```
      filter: {
        ':gt:name': '',  // workaround to filter on resources that have a 'name' attribute
      }
```

Ik zou er zeker wel als comment laten bijstaan dat we die filtering doen omwille van de organisaties die via ACM/IDM komen.